test/unit/logic/block_reward.js

```diff
@@ -121,37 +121,49 @@ describe('BlockReward @slow', () => {
 
 		it('when height == (milestoneFour * 2) should return 4', () => {
 			return expect(
```

Comment:
Those tests in this file are not good. `blockReward.calcMilestone` is accepting `height` as parameter, not a `Bignum` instance. Also, expectations are wrong, returning false-positives (`expect(someting);`).

Reply:
Please @diego-G open a separate issue for fixing the unit tests with the `equals` pointless expectation checks. Please specify which functions need to be covered.